src/index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'roundom' {
+  export default function roundom<T>(array: T[], seed?: number): T[];
```

Comment:
I never wrote a types file, but shouldn't first argument be Array<number>? Return value likewise?

Reply:
That is correct. :sweat_smile: I used a type variable out of habit when there is an array involved, but since it must always be an array of numbers the type variable is the wrong thing to use. I'll fix it now.